### DISCUSSION TITLE
Current Jellyfin Progress syncing status

### DISCUSSION THREAD

**Discussion by JackBailey** [arabcoders/watchstate#408] [General] [answered]
Hi, I've started using this project again as plex is playing up for me again, project is great.

What's the current status of the Jellyfin current progress syncing? README mentions its when the PR is merged, is it just waiting on the next version of Jellyfin for it to be added, or is it meant to be working currently?

**Comment by arabcoders** [comment, marked as answer]
Hi, the code to support the feature already included in the codebase. However, As jellyfin 10.9.x not yet released it will not work.
